source/declarations.tex

```diff
@@ -8090,7 +8090,8 @@
 \indextext{class!linkage specification}%
 A C language linkage is ignored
 in determining the language linkage of
-class members and the
+class members,
+friend functions with a trailing \grammarterm{requires-clause}, and
```

Comment:
the trailing "the" got dropped accidentally

Reply:
Yes, thanks, my mistake when merging.